bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java

```diff
@@ -1244,8 +1244,6 @@ Options getOptions() {
             super(CMD_LISTBOOKIES);
             opts.addOption("rw", "readwrite", false, "Print readwrite bookies");
             opts.addOption("ro", "readonly", false, "Print readonly bookies");
-            opts.addOption("h", "hostnames", false,
```

Comment:
I think it is not a good idea to remove an option directly. I will suggest marking this as deprecated and remove this in 4.8.

Reply:
ok..I'll add it back but it will effectively be ignored. BTW I dont see any particular format for adding deprecated tag for command line option. I'll just add following comment for the 'hostnames' option -

"//@deprecated 'rw'/'ro' option prints both hostname and ip, so this option is not needed anymore"

Reply:
oh there is not particular format. I was suggesting adding this to the description

"Also print hostname of the bookie" =>

"Also print hostname of the bookie (deprecated, 'rw'/'ro' option print both hostname and ip)"